extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSConfig.java

```diff
@@ -13,65 +13,76 @@
      * it is interpreted as a regular expression.
      */
     @ConfigItem
     @ConvertWith(TrimmedStringConverter.class)
     public Optional<List<String>> origins = Optional.empty();
 
     /**
-     * HTTP methods allowed for CORS
+     * Specifies the HTTP methods allowed for CORS requests.
      *
-     * Comma separated list of valid methods. ex: GET,PUT,POST
-     * The filter allows any method if this is not set.
+     * Provide a comma-separated list of valid HTTP methods, such as `GET,PUT,POST`.
+     * If not set, the filter allows any HTTP method by default.
      *
-     * default: returns any requested method as valid
+     * Default: Any requested HTTP method is allowed.
      */
     @ConfigItem
     @ConvertWith(TrimmedStringConverter.class)
     public Optional<List<String>> methods = Optional.empty();
 
     /**
-     * HTTP headers allowed for CORS
+     * Specifies the HTTP headers allowed for CORS requests.
      *
-     * Comma separated list of valid headers. ex: X-Custom,Content-Disposition
-     * The filter allows any header if this is not set.
+     * Provide a comma-separated list of valid headers, such as `X-Custom,Content-Disposition`.
+     * If not set, the filter allows any header by default.
      *
-     * default: returns any requested header as valid
+     * Default: Any requested HTTP header is allowed.
      */
     @ConfigItem
     @ConvertWith(TrimmedStringConverter.class)
     public Optional<List<String>> headers = Optional.empty();
 
     /**
-     * HTTP headers exposed in CORS
+     * Specifies the HTTP headers exposed in CORS responses.
      *
-     * Comma separated list of valid headers. ex: X-Custom,Content-Disposition
+     * Provide a comma-separated list of headers to expose, such as `X-Custom,Content-Disposition`.
      *
-     * default: empty
+     * Default: No headers are exposed.
      */
     @ConfigItem
     @ConvertWith(TrimmedStringConverter.class)
     public Optional<List<String>> exposedHeaders = Optional.empty();
 
     /**
-     * The `Access-Control-Max-Age` response header value indicating
-     * how long the results of a pre-flight request can be cached.
+     * Sets the `Access-Control-Max-Age` response header value.
+     *
+     * Specifies how long the browser can cache the results of a preflight request.
+     * Accepts durations in `java.time.Duration` format, such as `60` for sixty seconds, and `PT1H1S` for 1 hour and one second.
+     * See the <a href="https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html#parse(java.lang.CharSequence)">
+ * Duration#parse() Java API documentation</a> for details.
+     *
+     * Simplified formats:
+     * - A number represents seconds.
+     * - A number followed by `MS` represents milliseconds.
+     * - A number followed by `H`, `M`, or `S` is prefixed with `PT` (hours, minutes, or seconds).
+     * - A number followed by `D` is prefixed with `P` (days).
```

Comment:
@rolfedh IMHO explaining here how duration must be configured here is not necessary, given you link to Duration above

Reply:
Thanks so much! I'm updating the document accordingly.

I noticed that the original CORS guide states: "If the value is a number followed by ms, it represents time in milliseconds." However, I couldn't find any reference to ms in the [Java Duration API documentation](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html).

Do you think this might be a Quarkus-specific option, or could it be an error? Should I omit ms for now to avoid confusion?